api/.env

```diff
@@ -0,0 +1,24 @@
+# These are the enviromental variabes availabe for the api.  
+# Use this file for internal development or when launching the docker container locally.
+# For example docker run --env-file .env -p  <container_id>
+
+# Applications Configuration
```

Comment:
missing the `ALLOWED_ORIGIN` setting, which should have a value of "*" so that CORS works

Reply:
Added the ALLOWED_ORIGIN=*